fineract-provider/src/main/resources/application.properties

```diff
@@ -37,6 +37,7 @@ fineract.tenant.description=${FINERACT_DEFAULT_TENANTDB_DESCRIPTION:Default Demo
 
 fineract.mode.read-enabled=${FINERACT_MODE_READ_ENABLED:true}
 fineract.mode.write-enabled=${FINERACT_MODE_WRITE_ENABLED:true}
+fineract.mode.batch-enabled=${FINERACT_MODE_BATCH_ENABLED:true}
```

Comment:
@hiteshprajapati I am not so sure about this property here... why is it introduced if it's not used anywhere in your changes here? And if it's used in a different place: are we sure we want to set this by default to true? From what I can see here I don't think this property needs to be added.